### DISCUSSION TITLE
Long macron style?

### DISCUSSION THREAD

**Discussion by TheKnightWho** [psb1558/Junicode-font#113] [Ideas]
Hi - would it please be possible to add another stylistic set to cv84 for a wider macron? This would be useful for the letter m (as in Statutes of the Realm).

**Comment by psb1558** [comment]
Can you show me an example of what you want? I'm wondering if it would be preferable to automate it so that m always gets a wider macron. But if you're thinking of a bar the width of the letter, that's another solution.

**Reply by TheKnightWho** [reply]
The best example is actually with a tilde (which I should have mentioned as well). Vol. 1. p. lxiii of Statutes of the Realm, second table, middle column. Compare 'omnes' and 'Episcopis'. You can see that the 'm' has the wider tilde also used on 'co' in the third column.

![image](https://user-images.githubusercontent.com/6895210/156437233-3bb687c0-1ea6-4366-8ee4-5e90d1742e17.png)

**Reply by TheKnightWho** [reply]
Found an example of the macron:

![image](https://user-images.githubusercontent.com/6895210/156445912-3d9aa127-bb40-4796-abb5-c4bc89226906.png)
